app/serializers/operatingsystem_serializer.rb

```diff
@@ -3,7 +3,7 @@ class OperatingsystemSerializer < ActiveModel::Serializer
   embed :ids, include: true
 
   attributes :id, :name,
-             :description, :major, :minor, :family, :release_name
+             :description, :major, :minor, :family, :release_name,
```

Comment:
Also fixed a small typo of a missing comma which was breaking the webui from running